src/Geolocator.Plugin.Abstractions/IGeolocator.cs

```diff
@@ -62,14 +62,28 @@ public interface IGeolocator
         /// <returns>Position</returns>
         Task<Position> GetPositionAsync(TimeSpan? timeout = null, CancellationToken? token = null, bool includeHeading = false);
 
-		/// <summary>
-		/// Start listening for changes
-		/// </summary>
-		/// <param name="minimumTime">Time</param>
-		/// <param name="minimumDistance">Distance</param>
-		/// <param name="includeHeading">Include heading or not</param>
-		/// <param name="listenerSettings">Optional settings (iOS only)</param>
-		Task<bool> StartListeningAsync(TimeSpan minimumTime, double minimumDistance, bool includeHeading = false, ListenerSettings listenerSettings = null);
+        /// <summary>
+        /// Gets position async and reverse geocode
+        /// </summary>
+        /// <returns>Address of the current position</returns>
+        Task<Address> ReverseGeocodeCurrentLocation();
```

Comment:
I don't think we should combine the two of them together at all. I say make them pass it in.